app/common.py

```diff
@@ -236,6 +240,14 @@ def copytree_identical(src: Path, dst: Path):
             dirs.clear()  # stop iterate the subfolders of this dir
             continue
 
+        # NOTE(20220803): os.walk now lists symlinks pointed to dir
+        # in the <dirs> tuple, we have to handle this behavior
+        for _dir in dirs:
+            _src_dir = _cur_dir_on_src / _dir
+            _dst_dir = _cur_dir_on_dst / _dir
+            if (not _src_dir.is_symlink()) and _dst_dir.is_symlink():
+                _dst_dir.unlink()
```

Comment:
Is this block already done in L188?

Reply:
L188~ is iterating under the src folder, while L243~ is under through the dst folder.